alws/utils/github_integration_helper.py

```diff
@@ -101,12 +101,12 @@ async def set_build_id_to_issues(
     issues: list,
     build_id: str,
 ):
+    url = urljoin(settings.frontend_baseurl, f"build/{build_id}")
```

Comment:
Add `+ '/'` for a case when a first argument is not ending to `/`. The concatenation will be wrong in that case.
Same below.

Reply:
Fixed by putting validation onto `frontend_baseurl` which adds `/` in the end automatically